service/app.js

```diff
@@ -0,0 +1,332 @@
+import express from 'express'
+import mongoose from 'mongoose'
+import cors from 'cors'
+
+import { Deck } from './models/Deck.js'
+import { User } from './models/User.js'
+
+const app = express()
+const port = 8000
+
+// Connect to MongoDB
+
+const connectionString = `mongodb+srv://krishana:krishana@cluster0.m3dzd.mongodb.net/Cards?retryWrites=true&w=majority`
+try {
+  await mongoose.connect(connectionString)
+} catch (err) {
+  console.log('error ', err)
+}
+
+// Middleware
+
+const exampleMiddleware = (req, res, next) => {
+  console.log('example middleware')
+  next()
+}
+
+app.use(cors())
+app.use(express.json())
+app.use(exampleMiddleware)
+
+// Routes
+
+app.get('/', (req, res) => {
+  res.send('Hello, world!')
+})
+
+// Create user
+app.post("/createUser", async (req, res) => {
```

Comment:
You've failed to understand the naming process for routes, but only by a little. The path ("/createUser") should not have verbs, only nouns. The 'create' part of that is implied by using POST as a method. The path should just be '/user', because we want to POST (aka create) a user. This applies to all of your routes.

Reply:
It has been updated, Thank you.